UPGRADE-3.x.md

```diff
@@ -1,6 +1,14 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.17.3 to 3.x
+=========================
```

Comment:
I think we generally write 3.x to 3.x.

Because if it's from 3.17.3 the next version is necessarily 3.18

Reply:
I will leave it with 3.x to 3.x